src/integration-tests/introspector/introspectTest.sh

```diff
@@ -304,6 +320,11 @@ function deployCustomOverridesConfigMap() {
      ${SCRIPTPATH}/util_subst.sh -g ${filname} ${cmdir}/${bfilname}  || exit 1
   done
 
+  # We don't use overrides for MII
```

Comment:
If we don't use custom overrides for me, than this function should never be called in the first place.

Reply:
It won't work if it is not called. The operator has default config overrides which requires version.txt.  That's why I only clean up others to avoid duplicating the block and if conditions ..

Reply:
i don't understand.  The operator's 'internal overrides' have no version.txt.  The custom overrides config map has that file, and custom overrides config map should not be set up for the MII path.  In fact, if you add that to the MII domain resource, the domain deployment is expected to fail (since custom overrides aren't supported with MII).

Reply:
Yes, we do `override--version.txtt`  If I don't call this it won't creates it in the location and the introspect failed. 

[2020-04-03T16:27:43.390 UTC][][SEVERE] Error: Exiting test due to domainValid from introspecting domain is false!

@[2020-04-03T16:27:39.862 UTC][introspectDomain.py:277][FINE] Printing file /tmp/introspect/domain1/topology.yaml
>>>  /tmp/introspect/domain1/topology.yaml
domainValid: false
validationErrors:
- "Error, Required file, '/weblogic-operator/config-overrides/version.txt', does not exist"

>>> EOF

Reply:
OK. It turns out the test always mounts the config map even though it isn't used, and the introspector takes a different action when it sees this mount.  This is a side effect of the fact that the test doesn't actually run the operator.  I guess this is a fine work-around - we're not trying to emulate the operator perfectly.

Reply:
Resolving.